app/views/static_pages/start_page.html.erb

```diff
@@ -0,0 +1,36 @@
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-xl">
+        <%= t('student_loans.journey_name') %>
+      </h1>
+
+      <p class="govuk-body">
+        Claim student loan repayments if you:
+      </p>
+
+      <ul class="govuk-list govuk-list--bullet">
+        <li>
+          teach biology, chemistry, physics, computer science or languages (not including English)
+          for over half your working hours, including your planning, preparation and assessment (PPA) time
+        </li>
+        <li>have been teaching for 1 to 5 years</li>
+        <li>teach in certain places - <a href="https://www.gov.uk/guidance/teachers-student-loan-reimbursement-guidance-for-teachers-and-schools#eligibility-criteria">check which places are eligible</a></li>
+        <li>received qualified teacher status in or after the 2013 to 2014 academic year</li>
+      </ul>
+
+      <h2 class="govuk-heading-m">Before you apply</h2>
```

Comment:
Should this be `govuk-heading-l`? Isn't that the usual class for `h2`s?

Reply:
Not sure. I'll defer to @titlescreen on that one, I nabbed it wholesale from the repo.

Reply:
Had a look on MPS prototype and https://www.gov.uk/check-mot-history to compare and see what others do and they are using the `govuk-heading-m` font size. The use of headings is sequential so we are good form accessibility point of view, I just don't think the before you start information needs to stand out enough to justify the `govuk-heading-l`.